charts/athens-proxy/README.md

```diff
@@ -0,0 +1,150 @@
+
+```console
+helm install ./charts/proxy -n athens --set service.type=NodePort
+```
+
+### Ingress Resource
+
+The chart can optionally create a Kubernetes [Ingress Resource](https://kubernetes.io/docs/concepts/services-networking/ingress/#the-ingress-resource) for you as well. To enable this feature, set the `ingress.enabled` resource to true. 
+
+```console
+helm install ./charts/proxy -n athens --set ingress.enabled=true
+```
+
+Further configuration values are available in the `values.yaml` file:
+
+```yaml
+ingress:
+  enabled: false
+  # provie key/value annotations
+  annotations:
+  # Provide an array of values for the ingress host mapping
+  hosts:
+  # Provide a base64 encoded cert for TLS use 
+  tls: 
+```
+
+### Replicas
+
+By default, the chart will install Athens with a replica count of 1. To change this, change the `replicaCount` value:
+
+```console
+helm install ./charts/proxy -n athens --set replicaCount=3
+```
+
+### .netrc file support
+
+A `.netrc` file can be shared as a secret to allow the access to private modules.
+The secret must be created from a `netrc` file using the following command (the name of the file **must** be netrc):
+
+```console
+kubectl create secret generic netrcsecret --from-file=./netrc
+```
+
+In order to instruct athens to fetch and use the secret, `netrc.enabled` flag must be set to true:
+
+```console
+helm install ./charts/proxy -n athens --set netrc.enabled=true
+```
+
+### GoCenter as upstream proxy server
+
+[GoCenter](https://gocenter.jfrog.com) is a public central repository of Go modules which provides repeatable and fast module resolution. GoCenter automatically processes and validates Go projects directly from source code repositories into immutable, versioned modules, making them freely available to the Go developer community. In this way, it empowers the Go community to reliably generate reproducible builds of their Go applications.
+
+```yaml
+goCenter:
+  # Set https://gocenter.io as upstream proxy
+  enabled: true
+  url: "https://gocenter.io"
+```
+
+### Private git servers over ssh support
+
+One or many git servers are added to `sshGitServers`, and the corresponding config files (git config and ssh config) and ssh keys will be created. Athens then will use these configs and keys to download the source from the git servers.
+
+```yaml
+sshGitServers: 
+  ## Private git servers over ssh
+  ## to enable uncomment lines with single hash below
+  ## hostname of the git server
+  - host: git.example.com
+    ## ssh username
+    user: git
+    ## ssh private key for the user
+    privateKey: |
+      -----BEGIN RSA PRIVATE KEY-----
+      ...
+      -----END RSA PRIVATE KEY-----
+    ## ssh port
+    port: 22
+```
```

Comment:
@rimusz a lot of this file starting with "Deploy Athens" is also in https://docs.gomods.io/install/install-on-kubernetes/. The added parts look like the "GoCenter as upstream proxy server" and "Private git servers over ssh support" sections. Could you remove everything from "Deploy Athens" and below, add the new parts to https://docs.gomods.io/install/install-on-kubernetes/, and link to that install documentation in here?